Nimble/src/app/app.component.css

```diff
@@ -1,31 +1,42 @@
 header {
   position: relative;
-  background-color: #EBEBEB;
+  background-color: #FFFFFF;
   height: 110px;
 }
 
 body {
   position: relative;
+  font-family: "Open Sans";
   height: 330px;
   width: 600px;
   padding-left: 25%;
 }
 
+h1, h2, h3, h4, h5, h6 {
+  font-family: "Open Sans";
```

Comment:
You can change the font of everything with
```
:root {
  font-family: "Open Sans";
}
```
in [styles.css](https://github.com/heagleucsc/Nimble/blob/master/Nimble/src/styles.css)

Reply:
oh okay cool, thanks!